### ISSUE TITLE
Ability to delete an island

### ISSUE THREAD

**Issue by 3dphoto** [s1pierro/Papier#13]
Within the 2D paper area provide user with the ability to delete while **hover over shape** and **pressing delete** on the keyboard. I’ve thought about the user having the common [X] in the corner but you will probably come to find this will get annoying for the user trying to work efficiently plus you don’t want to bombard a user with extra noise while using a prompt that says [ Are you sure? ]

**Novice users**
[ X ]
I would though put the [ X ] in the top right of the 2D paper cut-out box with a prompt for 
[ Are you sure you want to delete? ]

**Advanced users**
You could create a **setting** in the **hamburger menu** to set for always **delete on hover,** when a **user(s) presses** the **delete key** from the keyboard

So when a **user presses** the **[ X ] + OK button** this would **clear** only what has been **selected** from the **2D paper cut-out(s) +** the **color/selection** on the **3D surface** of the model
![image](https://user-images.githubusercontent.com/28678758/41204670-8a645884-6c9c-11e8-85d8-845819acd570.png)

To give you, if that is you decide to remove all islands from A4
![image](https://user-images.githubusercontent.com/28678758/41204674-a02fe49e-6c9c-11e8-8976-769755de68d2.png)

**Comment by s1pierro** [comment]
yes, i'm thinking about a dynamic trash icon that would appear when an island is selected. Erasing all island should be placed in the settings view;

**Comment by s1pierro** [comment]
I'm agree with you about the confirmation alert
